addons/calendar/models/calendar_event.py

```diff
@@ -858,12 +857,25 @@ def action_open_calendar_event(self):
     def action_sendmail(self):
         email = self.env.user.email
         if email:
-            for meeting in self:
-                meeting.attendee_ids._send_mail_to_attendees(
-                    self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
-                )
+            self._send_template_to_attendees('calendar.calendar_template_meeting_invitation', self.attendee_ids)
         return True
 
+    @api.model
+    def _send_template_to_attendees(self, template_name, attendees, raise_if_not_found=False):
```

Comment:
We don't *need* the method, but it's a convenience as it allows me to have a docstring to describe the behavior and avoids having long lines just to send a template with a specific common parameter.

I guess it's a bit redundant with having a constant, but I like named constants for arbitrary limits :/

Reply:
I like the constant too!
The point is, if the alternative is adding 3 lines in two different places (both using this constant), it'll be more readable (and shorter) than these 16 lines ;) That said, I see it's three times now, so maybe it's not feasible...
If `send_mail_to_attendees` and `send_template_to_attendees` (which should otherwise be `send_mail_to_attendees_with_template` or something) can be merged, probably even better.

Reply:
Why not adding the feature directly into ``_send_mail_to_attendees`` ? It means having a 3 level parameter (force send, use parameter, never force send), not sure how to have something clean but it would avoid yet another method.